src/providers/providers/index.ts

```diff
@@ -146,12 +146,26 @@ export const BINANCECHAINWALLET: IProviderInfo = {
   check: "isBinanceChainWallet"
 };
 
+/**
+ * @deprecated Use CoinbaseWalletSdk
```

Comment:
not sure `WALLETLINK` is necessary for compatibility. if this is not being used externally, i think we would just remove this.

Reply:
this is required